lib/model/special_triggers.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:math';
+
+import 'package:treasure_of_the_high_seas/model/resource.dart';
+
+import 'hand.dart';
+
+bool checkMutiny(Hand playerHand) {
+
+  // Mutiny next card if resource count over 15
+  return playerHand.cards.length > 15;
+
+}
+
+bool checkNavyRaid(Hand playerHand){
+
+  // Navy Raid next card if not Mutiny and >4 infamy and you toss a heads
+  // TODO - ran into same problem with testing this
+  Random random = new Random();
+  return playerHand.countResource(Resource.INFAMY) > 4 && random.nextBool();
```

Comment:
Ah more random fun. Perhaps we make our `ListShuffler` more generic and rename it to something like `Randomiser`. We already have one of these in the `GameState`, so if we passed the full state to this method rather than just the hand we could access it and use it to flip a coin (using a new method we'd add, which would do `random.nextBool()`. 

Then we could have a `FakeRandomiser` which we construct with a boolean of `true` or `false` and just returns that for the `flipACoin` method, which we'd use in tests.

Reply:
Agreed, done.